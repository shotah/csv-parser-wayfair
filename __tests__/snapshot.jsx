import React from 'react';
import { render } from '@testing-library/react';
import { RecoilRoot } from 'recoil';
import Home from '../pages/index';

it('renders homepage unchanged', async () => {
  const { asFragment } = render(
    <RecoilRoot>
      <Home />
    </RecoilRoot>
  );
  expect(asFragment()).toMatchSnapshot();
});
