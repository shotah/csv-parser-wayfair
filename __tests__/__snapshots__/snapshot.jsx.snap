// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<DocumentFragment>
  <div
    class="md-container container"
  >
    <div
      class="container"
    >
      <h1>
         Welcome to Favicon Finder! 
      </h1>
      <h5>
        Designed to enable you to quickly find all the favicons!
      </h5>
      <br />
      <button
        class="btn btn-primary"
        type="button"
      >
        Upload CSV
      </button>
        
      <button
        class="btn btn-primary"
        disabled=""
        type="button"
      >
        Get FavIcons
      </button>
        
      <button
        class="btn btn-primary"
        disabled=""
        type="button"
      >
        Export CSV
      </button>
        
      <button
        class="btn btn-primary"
        disabled=""
        type="button"
      >
        Clear
      </button>
      <br />
      <br />
      <h4>
         Please upload a CSV. 
      </h4>
    </div>
  </div>
</DocumentFragment>
`;
